### ISSUE TITLE
perfectly

### ISSUE THREAD

**Issue by Bronxanarchy** [mobb111/ruby-brainwalet-destroyer#2]
work perfectly, cogratulations,

But I think,

no need

**Comment by mobb111** [comment]
when balance is >0 its mean that balance is positive and will be saved in ruby.txt file

**Comment by mobb111** [comment]
i can do that i know what u meaning i will change this in few days :)

**Comment by mobb111** [comment]
this is published look in my other generators the anogen is like that

**Comment by Bronxanarchy** [comment]
yes, but anogen question at sha256 passphare

**Comment by mobb111** [comment]
salt for sha256 is password they added to secure bitcoin remember that if u not use it in recovery leave that emty

**Comment by Bronxanarchy** [comment]
Yes, i know, i have wordlist more, i think try only wordlist and balance

29 Eyl 2018 Cmt 17:26 tarihinde mobb <notifications@github.com> şunu yazdı:

**Comment by mobb111** [comment]
https://www.youtube.com/watch?v=AlH_4E3sd8c

**Comment by mobb111** [comment]
offline harvester

**Comment by Bronxphoneix** [comment]
perfectly, its sale I think

mobb <notifications@github.com>, 30 Eki 2018 Sal, 14:50 tarihinde şunu
yazdı:

**Comment by mobb111** [comment]
i publish new one better one try it
